wordpress-mobile/WordPress-Android
# Implement EDIT_HOMEPAGE QS action and tests

Fixes #13877 

This PR adds functionality to the edit homepage dynamic card. It now scrolls the app to the Quick actions, shows a focus point and on click triggers the previously implemented logic (with the sticky event). The only change to the previous logic is that the `complete` action is now passed to the QuickStartRepository from the pages screen (if the flag is enabled). Don't mind the editor crash in the video. 

To test:
- Choose a site with a "Homepage" selected
- Make sure the my site improvements flag is turned on and the app is restarted
- Click on the "Edit homepage" card
- Notice the app scrolls to the quick actions card
- Notice the focus point on the "Pages" action
- Click on the "Pages" action
- Notice the focus point on the Homepage
- Click on the Homepage
- Go back to My Site
- Notice the "Edit Homepage" action is now completed
- Switch to a site without a Homepage
- Notice the "Edit Homepage" task is autocompleted

![edit_homepage_task](https://user-images.githubusercontent.com/1079756/106629495-30db6700-657b-11eb-9011-9a02148a66bc.gif)


PR submission checklist:
